spec/integration/halBoard.json

```diff
@@ -65,7 +67,8 @@
 									"href": "/test/api/card/301/block",
 									"method": "PUT"
 								}
-							}
+							},
+							"_version": 1
```

Comment:
If that's the case, do we need to include `_version` on all of the embedded resources?